docs/commands.rst

```diff
@@ -5,6 +5,9 @@ Commands for maintaining a language repository
 This document contains instructions to guide you on managing your language
 repository, hence helping you translate Python's documentation to your language.
 
+The :doc:`sample workflow's <workflows>` Transifex workflow is an automated
+alternative to this guide.
```

Comment:
Sounds weird to me "The sample workflow's Transifex workflow ". How about (were 'workflows' ref is replaced with GitHub Worfklow):

```suggestion
See :doc:`workflows`, an automated alternative to this guide in GitHub Actions.
```